### DISCUSSION TITLE
Fixing the DNS resolv.conf issue

### DISCUSSION THREAD

**Discussion by rusty-snake** [netblue30/firejail#5607] [General]
A lot network connection issues are caused by `/etc/resolv.conf` being outdated or wrong in other ways.

Examples:

* #1889
* #3646
* #3649
* #4954
* #5010
* #5171
* #5746

Relates to:

* #5591
* #5608
* #5609
* #6400

When using options like `private-etc`, we bind mount `/etc/resolv.conf` into the sandbox. This is a problem if programs like NetworkManager/... make changes to `/etc/resolv.conf` by replacing rather than editing. (How do we handle the symlink case?).

This discussion is to find a concept to always properly propagate `/etc/resolv.conf` changes to the sandbox.

**Comment by e-pirate** [comment]
Just saying: came here from [#5010 ] to stay in tune. Currently using 0.9.64.4 cause later versions are not usable in my every day scenario.
